src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogBuildListener.java

```diff
@@ -317,16 +312,18 @@ public void onCompleted(Run run, @Nonnull TaskListener listener) {
 
             if (run instanceof WorkflowRun) {
                 RunExt extRun = getRunExtForRun((WorkflowRun) run);
-                long pauseDurationMillis = 0;
-                for (StageNodeExt stage : extRun.getStages()) {
-                    pauseDurationMillis += stage.getPauseDurationMillis();
+                if (extRun != null){
```

Comment:
Should we drop a log if the `extRun` is `null`?

Reply:
The `getRunExtForRun()` method that returns it will log `Error while getting RunExt` along with the exception.